pkg/controllers/federatedcluster/controller.go

```diff
@@ -63,6 +63,9 @@ const (
 	EventReasonHandleTerminatingClusterBlocked = "HandleTerminatingClusterBlocked"
 )
 
+// annotationForceDelete on a federated cluster means that cluster should be force delete by KubeAdmiral.
+var annotationForceDelete = common.DefaultPrefix + "force-delete"
```

Comment:
```suggestion
const AnnotationForceDelete = common.DefaultPrefix + "force-delete"
```
Please merge this with the const block above.

Reply:
done.